beachball.config.js

```diff
@@ -1,2 +1,16 @@
-require("./build/releasing/ts-node-register");
-module.exports = require("./build/releasing/index");
+module.exports = {
+    disallowedChangeTypes: ["major"],
+    ignorePatterns: [
+        ".ignore",
+        ".github/",
+        ".prettierrc",
+        ".vscode/",
+        "docs/**",
```

Comment:
@Ibrahimmaga @awentzel I *think* we'll want to leave off docs for things *within* the packages folder. Docs would include the API report here and we *definitely* want to throw on changed files if that is changed and hasn't been included. Any docs within sites are excluded with the change below, but I don't think we want to ignore packages which have a `docs/` folder due to the API report.